### ISSUE TITLE
Indian Billing GST Tax Rules[Feature Request] 

### ISSUE THREAD

**Issue by swapnildmahajan** [boxbilling/boxbilling#1443]
### What is the idea?

In India, GST Billing is used where home state is being charged 2 Taxes SGST and CGST at same Percentage and other than home state Only one tax i.e. IGST is charged, 

Current tax rule doesn't allow us to apply 2 tax rule at a time

### Why would this be beneficial

In India without GST no one will be able to raise invoice...

**Comment by SrinathReddy123** [comment]
Can I contribute to this feature request?

**Comment by evrifaessa** [comment]
> Can I contribute to this feature request?

Of course. But we would recommend you to switch to FOSSBilling instead. We're the latest team that were maintaining BoxBilling for the last 2 years. We've recently had to move to a new organization. For more information, visit our [website](https://fossbilling.org) and join our [Discord server](https://fossbilling.org/discord).
